README.md

```diff
@@ -6,35 +21,35 @@
 [![Code Climate](https://codeclimate.com/github/khiav223577/active_model_cachers/badges/gpa.svg)](https://codeclimate.com/github/khiav223577/active_model_cachers)
 [![Test Coverage](https://codeclimate.com/github/khiav223577/active_model_cachers/badges/coverage.svg)](https://codeclimate.com/github/khiav223577/active_model_cachers/coverage)
 
-Provide cachers to the model so that you could specify which you want to cache. Data will be cached at `Rails.cache` and also at application level via `RequestStore` to cache values between requests. Cachers will maintain cached objects and expire them when they are changed (including create, update, destroy, and even delete).
+ActiveModelCachers provides cachers to models and allows the users to specify what needs to be cached. The data will be cached at `Rails.cache` and also at application level via `RequestStore`, to cache values between requests. The cachers will maintain cached objects and expire them when they are changed (e.g. created, updated, destroyed, or deleted).
 
-- [Multi-level Cache](#multi-level-cache)
```

Comment:
This line is missing

Reply:
Sure, I will add the line. Can we also add a verb to it? The rest of the bullet-pointed items start with verbs which make them easier to understand, but this item is only referring to another heading in the readme. 
I think it would be better if we say something like "caching is multileveled" or "supports Multi-level Cache" (whichever you think has a closer meaning to what you mean to say). This will also make the bullet-pointed items consistent with each other.

Reply:
`caching is multileveled` is great. This makes it more understandable 👍

Reply:
I also believe "caching is multileveled" is easier to understand but I just realized that "ActiveModelCachers" is the subject of the other 3 sentences so the sentence will look a bit inconsistent with the others if we use it. Do you think "uses multiple levels of cache" is also close to what you mean to say?